android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy

```diff
@@ -339,8 +337,7 @@ Depends: ${dependencies}
     args.add(androidRepo.absolutePath)
     args.add("--out")
     args.add(outputRepo.absolutePath)
-    // FIXME: figure that one out
-    args.add("fr.epardaud.testjava.testjava/1")
+    args.add(project.androidCeylon.mainModule)
```

Comment:
What happens if there's more than one module in `source`? Do we have to jigsaw 'em all, or can the tool detect which module is the "main" one?